<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 33,79);
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">Qframe{
background-color: rgb(191, 186, 43);}

QpushButton{
border-radius:18px;
background-color: rgb(255, 103, 43);}

QpushButton:hover{
background-color: rgb(0, 85, 255);}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>803</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bt_minimize">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>IMAGENES/mi.png</normaloff>IMAGENES/mi.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_normal">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>IMAGENES/ma.png</normaloff>IMAGENES/ma.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_maximize">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>IMAGENES/ma.png</normaloff>IMAGENES/ma.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_close">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>IMAGENES/x.png</normaloff>IMAGENES/x.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(30, 30, 30);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>401</width>
                 <height>61</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);
font: 26pt &quot;Bauhaus 93&quot;;</string>
               </property>
               <property name="text">
                <string>GRUPO 08</string>
               </property>
              </widget>
              <widget class="QComboBox" name="cb_list_ports">
               <property name="geometry">
                <rect>
                 <x>300</x>
                 <y>140</y>
                 <width>150</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">QComboBox{
	border: 3px solid rgb(255,255,255);
	border-radius: 5px;
	min-width: 6em;
	color: rgb(255,255,255);
	font: 87 10pt	&quot;Arial Black&quot; ;
}

QComboBox::drop-down{
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px;
	border-left-width:3px;
	border-left-color: rgb(255,255,255);
	border-left-style: solid;
	border-top-right-radius: 4px;
	border-bottom-right-radius: 4px;
}

QComboBox QAbstractItemView{
	background: #2c2c2c;
	selection-background-color: #ff0037;
	color: rgb(255,255,255);
}</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>180</x>
                 <y>500</y>
                 <width>100</width>
                 <height>100</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>100</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>360.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
              <widget class="QLabel" name="label_3">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>210</y>
                 <width>231</width>
                 <height>61</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Comic Sans MS&quot;;</string>
               </property>
               <property name="text">
                <string>VELOCIDAD (BAUNDS)</string>
               </property>
              </widget>
              <widget class="QComboBox" name="cb_list_baudrates">
               <property name="geometry">
                <rect>
                 <x>300</x>
                 <y>230</y>
                 <width>150</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">QComboBox{
	border: 3px solid rgb(255,255,255);
	border-radius: 5px;
	min-width: 6em;
	color: rgb(255,255,255);
	font: 87 10pt	&quot;Arial Black&quot; ;
}

QComboBox::drop-down{
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 15px;
	border-left-width:3px;
	border-left-color: rgb(255,255,255);
	border-left-style: solid;
	border-top-right-radius: 4px;
	border-bottom-right-radius: 4px;
}

QComboBox QAbstractItemView{
	background: #2c2c2c;
	selection-background-color: #ff0037;
	color: rgb(255,255,255);
}</string>
               </property>
              </widget>
              <widget class="QPushButton" name="bt_connect">
               <property name="geometry">
                <rect>
                 <x>150</x>
                 <y>310</y>
                 <width>171</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;Vineta BT&quot;;
background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>CONECTAR</string>
               </property>
              </widget>
              <widget class="QPushButton" name="bt_disconnect">
               <property name="geometry">
                <rect>
                 <x>120</x>
                 <y>380</y>
                 <width>231</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;Vineta BT&quot;;
background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>DESCONECTAR</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_4">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>120</y>
                 <width>231</width>
                 <height>61</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Comic Sans MS&quot;;</string>
               </property>
               <property name="text">
                <string>PUERTO SERIAL</string>
               </property>
              </widget>
              <widget class="QPushButton" name="bt_update">
               <property name="geometry">
                <rect>
                 <x>120</x>
                 <y>450</y>
                 <width>231</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;Vineta BT&quot;;
background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>ACTUALIZAR</string>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <widget class="QFrame" name="camara_frame">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(30, 30, 30);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <widget class="QLabel" name="label_5">
                 <property name="geometry">
                  <rect>
                   <x>30</x>
                   <y>20</y>
                   <width>231</width>
                   <height>61</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
font: 87 12pt &quot;Comic Sans MS&quot;;</string>
                 </property>
                 <property name="text">
                  <string>PROCESO:</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_6">
                 <property name="geometry">
                  <rect>
                   <x>40</x>
                   <y>90</y>
                   <width>231</width>
                   <height>61</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Comic Sans MS</family>
                   <pointsize>10</pointsize>
                   <weight>10</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Comic Sans MS&quot;;</string>
                 </property>
                 <property name="text">
                  <string>1. GUARDAR FOTO:</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_7">
                 <property name="geometry">
                  <rect>
                   <x>40</x>
                   <y>160</y>
                   <width>231</width>
                   <height>61</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Comic Sans MS</family>
                   <pointsize>10</pointsize>
                   <weight>10</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Comic Sans MS&quot;;</string>
                 </property>
                 <property name="text">
                  <string>2. PROCESAR FOTO:</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_8">
                 <property name="geometry">
                  <rect>
                   <x>40</x>
                   <y>230</y>
                   <width>231</width>
                   <height>61</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Comic Sans MS</family>
                   <pointsize>10</pointsize>
                   <weight>10</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Comic Sans MS&quot;;</string>
                 </property>
                 <property name="text">
                  <string>3. CALCULAR DISTANCIA:</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="save_photo">
                 <property name="geometry">
                  <rect>
                   <x>330</x>
                   <y>110</y>
                   <width>93</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;Vineta BT&quot;;
background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="procesing">
                 <property name="geometry">
                  <rect>
                   <x>330</x>
                   <y>180</y>
                   <width>93</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;Vineta BT&quot;;
background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="calculate">
                 <property name="geometry">
                  <rect>
                   <x>330</x>
                   <y>250</y>
                   <width>93</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;Vineta BT&quot;;
background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_9">
                 <property name="geometry">
                  <rect>
                   <x>40</x>
                   <y>370</y>
                   <width>231</width>
                   <height>61</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Comic Sans MS</family>
                   <pointsize>10</pointsize>
                   <weight>10</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Comic Sans MS&quot;;</string>
                 </property>
                 <property name="text">
                  <string>5. MOSTRAR FOTO:</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_10">
                 <property name="geometry">
                  <rect>
                   <x>40</x>
                   <y>300</y>
                   <width>231</width>
                   <height>61</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Comic Sans MS</family>
                   <pointsize>10</pointsize>
                   <weight>10</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Comic Sans MS&quot;;</string>
                 </property>
                 <property name="text">
                  <string>4. CONECTAR :</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="base_datos">
                 <property name="geometry">
                  <rect>
                   <x>330</x>
                   <y>320</y>
                   <width>93</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;Vineta BT&quot;;
background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="mostrar_foto">
                 <property name="geometry">
                  <rect>
                   <x>330</x>
                   <y>380</y>
                   <width>93</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;Vineta BT&quot;;
background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
